### PR TITLE
[MIG] stock_picking_show_return

### PR DESCRIPTION
**Pull request by pedrobaeza**
# 
# Show returns on stock pickings

This module enables one tab in the pickings that contains returns made for that
picking.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
cc @Tecnativa

File: stock_picking_show_return/README.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================
+Show returns on stock pickings
+==============================
+
+This module enables one tab in the pickings that contains returns made for that
```
Comment (elicoidal) [inline stock_picking_show_return/README.rst]: This module adds one tab in the pickings form view to display the returns pickings related to the current one.

File: stock_picking_show_return/README.rst [hunk 2]
```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================
+Show returns on stock pickings
```
Comment (elicoidal) [inline stock_picking_show_return/README.rst]: s/Show/Display

File: stock_picking_show_return/views/stock_picking_view.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <record id="view_picking_form" model="ir.ui.view">
+            <field name="model">stock.picking</field>
+            <field name="inherit_id" ref="stock.view_picking_form"/>
+            <field name="arch" type="xml">
+                <notebook position="inside">
+                    <page string="Returns" attrs="{'invisible': [('returned_ids', '=', [(6, False, [])])]}">
```
Comment (elicoidal) [inline stock_picking_show_return/views/stock_picking_view.xml:10]: Why not call it "Returned Pickings"?

**Comment by elicoidal** [comment]
:+1:

File: stock_picking_show_return/views/stock_picking_view.xml (hunk 3 above)
Comment (pedrobaeza) [inline stock_picking_show_return/views/stock_picking_view.xml:10]: Because it's larger and as we are already in the picking form, it will be duplicated.

Comment (elicoidal) [inline stock_picking_show_return/views/stock_picking_view.xml:10]: OK

**Comment by pedrobaeza** [comment]
Texts changed and added tests

**Comment by lasley** [comment]
Code + functional 👍

**Comment by rafaelbn** [comment]
Tested un runbot, very nice :+1: 

This PR is ready to be merged as reviewed and 3 x :+1:

**Comment by pedrobaeza** [comment]
Merging then

**Pull request merged**